Cargo.lock

```diff
@@ -2538,7 +2538,7 @@ dependencies = [
  "chrono",
  "clap 4.2.5",
  "dirs 5.0.0",
- "futures",
+ "futures-util",
```

Comment:
Thanks for the contribution. But I don't thin we can reduce deps via change `futures` into `futures-util`:

diff of `futures` & `futures-util`'s dep:

```diff
20,24c36,45
< futures-channel = { path = "../futures-channel", version = "=0.4.0-alpha.0", default-features = false, features = ["sink"] }
< futures-executor = { path = "../futures-executor", version = "=0.4.0-alpha.0", default-features = false, optional = true }
< futures-io = { path = "../futures-io", version = "0.3.28", default-features = false }
< futures-sink = { path = "../futures-sink", version = "=0.4.0-alpha.0", default-features = false }
< futures-util = { path = "../futures-util", version = "=0.4.0-alpha.0", default-features = false, features = ["sink"] }
---
> futures-channel = { path = "../futures-channel", version = "=0.4.0-alpha.0", default-features = false, features = ["std"], optional = true }
> futures-io = { path = "../futures-io", version = "0.3.28", default-features = false, features = ["std"], optional = true }
> futures-sink = { path = "../futures-sink", version = "=0.4.0-alpha.0", default-features = false, optional = true }
> futures-macro = { path = "../futures-macro", version = "=0.4.0-alpha.0", default-features = false, optional = true }
> slab = { version = "0.4.2", optional = true }
> memchr = { version = "2.2", optional = true }
> futures_01 = { version = "0.1.25", optional = true, package = "futures" }
> tokio-io = { version = "0.1.9", optional = true }
> pin-utils = "0.1.0"
> pin-project-lite = "0.2.6"
```

The only thing added here is `futures-executor`. I think we can remove it via disable the `executor` future.

Reply:
Makes sense. As core requires `futures-executor` at `services-sftp` feature, it's might be better to use `futures` and handle features to reduce costs to maintain dependencies.

Reply:
> As core requires `futures-executor` at `services-sftp` feature

We can enable this only on `services-sftp` feature.

Reply:
Agree. We can omit default features as default and add `futures/executor` to `services-sftp`.

Reply:
Fixed.